src/models/source_code_unit.rs

```diff
@@ -240,33 +240,20 @@ impl SourceCodeUnit {
 
       // Process the parent
       // Find the rules to be applied in the "Parent" scope that match any parent (context) of the changed node in the previous edit
-      if let Some(edit) = self.get_edit_for_context(
-        *current_replace_range.start_byte(),
-        *current_replace_range.end_byte(),
-        rules_store,
-        &next_rules_by_scope[PARENT],
-      ) {
+      if let Some(edit) =
+        self.get_edit_for_ancestors(&current_replace_range, rules_store, &next_rules_by_scope)
+      {
         self.rewrites_mut().push(edit.clone());
-        debug!(
```

Comment:
Just to confirm, we are intentionally getting rid of this debug log line entirely, right?